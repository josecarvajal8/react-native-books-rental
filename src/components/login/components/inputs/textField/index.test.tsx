import React from 'react';
import renderer from 'react-test-renderer';
import TextField from '.';
const mockProps = {
    handleChange: () => null,
    handleBlur: () => null,
    handlerErrors: () => null,
    fieldId: 'test',
    label: 'test_field',
    values: {},
    disable: false
}
test('Render textField component correctly', () => {
    const textFieldCmp = renderer.create(<TextField {...mockProps} />).toJSON();
    expect(textFieldCmp).toMatchSnapshot();
});