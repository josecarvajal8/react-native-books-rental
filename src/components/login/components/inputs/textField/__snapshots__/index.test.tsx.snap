// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render textField component correctly 1`] = `
<TextInput
  allowFontScaling={true}
  editable={true}
  onBlur={[Function]}
  onChangeText={null}
  onFocus={[Function]}
  placeholder="test_field"
  rejectResponderTermination={true}
  style={
    Object {
      "borderColor": "#626565",
      "borderRadius": 108.69565217391305,
      "borderWidth": 3.6231884057971016,
      "height": 66.99776785714286,
      "padding": 10,
      "paddingHorizontal": 18.115942028985508,
      "paddingVertical": 14.888392857142858,
      "width": 543.4782608695652,
    }
  }
  underlineColorAndroid="transparent"
/>
`;
